PART_7/StrainTransmission/Iterate_and_job.py

```
@@ -0,0 +1,186 @@
+from pathlib import Path
+import subprocess
+import time
+def check_job_count():
+	output = subprocess.check_output(['squeue', '-u', 'umcg-sandreusanchez']).decode('utf-8')
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)